dss_datamover/master_application.py

```diff
@@ -1178,7 +1178,7 @@ def process_get_operation(master):
                 master.logger.error("###### DataIntegrity Test Status: FAILED ######")
 
     # Start Compaction
-    if not master.process_monitor_event.is_set() and "compaction" in params and params["compaction"]:
+    if not master.process_monitor_event.is_set() and ('compaction' not in config or config['compaction'] == 'yes'):
```

Comment:
this is not the right way you are supposed to do. Everything should be updated in the config section..

Reply:
Let the user say yes/no,  but we can convert it to true/false

Reply:
you clearly mentioned in our design meeting that you would like compaction to be yes/no. 

> Also why do you need yes/no. You can make it True/False and reduces extra code checks

Reply:
I think you misunderstood what I am trying to say. The user gives yes/no, but the  code stores it as True/False. Then you don't need to do ```if x == 'yes':```. Instead the code looks like ```if x:```

Reply:
> this is not the right way you are supposed to do. Everything should be updated in the config section..

I use argparse.SUPPRESS so that compaction is not set by deault if user doesnt specify anything in order to support overriding from both CLI and config standpoint. I can add a layer of logic in the config block to set compaction to yes if the option was not explicitly set.